content/en/blog/_posts/2024-08-13-Kubernetes-v1-31-Release.md

```diff
@@ -269,13 +269,15 @@ Please see [Kubernetes issue #125689](https://github.com/kubernetes/kubernetes/i
 
 #### Deprecation of `status.nodeInfo.kubeProxyVersion` field for Nodes ([KEP 4004](https://github.com/kubernetes/enhancements/issues/4004))
 
```

Comment:
We should mark at the start of the section that that the article was edited post publication.

Reply:
done